app/pages/sections/order-items/catalogue-solutions/order-item/flat/ondemand/manifest.json

```diff
@@ -0,0 +1,146 @@
+{
+  "questions": {
+    "price": {
+      "id": "price",
+      "mainAdvice": "What price have you agreed? (£)",
+      "unit": "{{ unit | lower }}",
```

Comment:
As the manifest is just a JSON object that we use to supply data to the template, templating has no effect here so we may as well just use a literal placeholder, e.g. `"unit description"`.

Reply:
Done